client/layout/guided-tours/style.scss

```diff
@@ -1,6 +1,7 @@
 .guided-tours__step {
 	position: fixed;
 	max-width: 410px;
+	width: 410px;
```

Comment:
Thanks for the pointer. I looked at this a bit more and have now moved the width setting from the `guided-tours__first-step` class up here. This solves the problem we had here (very narrow steps when little text present) and at the same time reuses what we'd already been using anyway.